rules.ini

```
@@ -119,6 +119,9 @@ RPGMaker[] = (?:^|/)rgss(?:104e|102j|102e|301|202e)\.dll$
 RPGMaker[] = (?:^|/)rp[tg]_rt\.exe$
 RPGMaker[] = (?:^|/)RPG_RT\.ini$
 RPGMaker[] = \.(?:rgssad|rvproj2|rgss3a|rgss2a)$
+S2_Engine_HD[] = \.(?:an|(?:f|m)t|s(?:c|k|m|o))2$
+S2_Engine_HD[] = (?:^|/)S2Configuration.exe
+S2_Engine_HD[] = (?:^|/)S2(?:Base(?:AI|Pack)|Core|Cutscene|DDSRes|DXInput|Editor|GamePack|Gui|Null(?:Audio|Physics|Render)|OalAudio|OglRender3|PhysxPhysics3|Resource)\.dll$
```

```suggestion
S2_Engine_HD[] = (?:^|/)S2Core\.dll$
```
From what i saw this seems to be enough, the .exe one matches the same app as this one and the extension one seems to bring a lot of false positive 

the only exception might be these 2  `an2 gm2` but apps that they match are either already matched by the dll one or looks to be false positives